tests/unit/test_ecdsa.cpp

```diff
@@ -48,7 +48,7 @@ void TestEcdsaVector(void)
 {
     // This case is derived from the test vector from RFC 6979 - section A.2.5 (for NIST P-256 and SHA-256 hash).
 
-    const uint8_t kKeyPairInfo[] = {
+    uint8_t kKeyPairInfo[] = {
```

Comment:
Why do we need to change this?

We should declare input as `const uint *` unless function/method is expected/allowed to change the content